src/models/blockchain/block/mod.rs

```diff
@@ -1283,13 +1294,14 @@ pub(crate) mod tests {
                 let duration = i as u64 * multiplier;
                 now += Timestamp::millis(duration);
 
-                let (block, _) = make_mock_block(&block_prev, Some(now), a_key, rng.random()).await;
+                let (block, _) =
+                    make_mock_block(network, &block_prev, Some(now), a_key, rng.random()).await;
 
                 let control = difficulty_control(
                     block.kernel.header.timestamp,
                     block_prev.header().timestamp,
                     block_prev.header().difficulty,
-                    None, // target_block_interval
+                    network.target_block_interval(), // target_block_interval
```

Comment:
```suggestion
                    network.target_block_interval(),
```

Reply:
removed comment in 610407ed2c288d61957af39cb84d34ae0da4e935